ddl/ddl_api.go

```diff
@@ -5777,8 +5777,35 @@ func buildPlacementSpecReplicasAndConstraint(replicas uint64, cnstr string) ([]*
 		}
 
 		constraints := []string{}
+		byteArr := bytes.Split([]byte(cnstr), []byte(","))
```

Comment:
Commas may be in the quotes.

Reply:
if commas may be in the quotes, it will parse error, right ?

Reply:
the code will deal with this situation
```
if len(byteArr[i]) == 0 {
	break
}
```